### DISCUSSION TITLE
2023 年样式适配

### DISCUSSION THREAD

**Discussion by xjtu-blacksmith** [xjtu-blacksmith/easymcm#18] [General]
粗略检查了一下 COMAP 官方发布的指南和文件模板，似乎较去年没有变动。由于我自己已经多年不参加比赛，希望各位用户能帮助使用模板，提出问题，收到意见后我会及时更正模板中的相关内容。

**Comment by vonJOKER** [comment]
就是有个奇怪的问题就是当我用引用的参考文献是bib样式时，他编译出来的一直是PAPER.bib和PAPER.bbl，也就是说我该如何使模板能够编译我自己导出的bib文件并且引用呢，感谢指教!（我用xe-->bib*2-->xe和latexmk分别编译过但都是同样的问题）
